extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/ui/UiResource.java

```diff
@@ -66,4 +65,11 @@ public IdResponseDto initiateTransfer(
     public TransferHistoryPage transferHistoryPageEndpoint() {
         return transferHistoryPageApiService.transferHistoryPage();
     }
+
+    @GET
+    @Path("pages/transfer-history-page/{transferProcessId}/asset")
```

Comment:
I think we should have "pages/transfer-history-page/transfer-processes/{transferProcessId}/asset" to respect REST's resource-based naming conventions